### ISSUE TITLE
Notebook 4

### ISSUE THREAD

**Issue by stemlock** [dlab-berkeley/ann_fundamentals#16]
1. Build ANN Classifier: We show how to build an ANN using Keras Sequential API, but make no mention of the Functional API for more customized architectures. Might be worthwhile including a link to external material as primer on Keras Functional API.
2. Load and Visualize Data: Consider splitting train data into train and val and then running training using the validation data set, and evaluate on the test set.
